app/src/main/res/layout/activity_partners.xml

```diff
@@ -22,26 +22,37 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
+    <com.google.android.material.textfield.TextInputLayout
```

Comment:
Why did you move the TextField out of the `LinearLayout`?  Do we actually still need the `LinearLayout` below (It only contains one element in your solution)?

Reply:
Because I need the layout constraint feature for correct placement of the search field. The `LinearLayout` acts as container of the included XML so that the list view can be aligned correctly in the current layout. Since the included XML is used in different layouts, I probably break some other layout. Another PR is required, to unify all the layouts where the list view is included.